tests/test_dataset_implementations/test_dataset_implementations.json

```diff
@@ -83,6 +83,22 @@
                     "c": "INTEGER"
                 }
             }
+        },
+        "d4": {
+            "data": {
+                "a": [0, 0, 0, 0, 0]
+            },
+            "schemas": {
+                "SparkDFDataset": {
+                    "a": "IntegerType"
+                },
+                "mysql": {
+                    "a": "INTEGER"
+                },
+                "mssql": {
```

Comment:
Added this test case as Trino is returning error inconsistently with stddev_samp.

I am ok with removing this if not needed